test/unit/helpers/xhr/CONSENT_REQUIRED.js

```diff
@@ -13,7 +13,6 @@ define({
         "clientId": "8WUqrTzUG9RyJt2C6Gmm",
         "_links": {
           "logo": {
-            "href": "https://example.com/logo.png",
```

Comment:
how does the response look like when there is no logo? What we have makes it kind of confusing I think:
```
"_links": {
    "logo": {
        "type": "image/png"
    },
```

I would have thought that the API will not return you a 'logo' node at all if there is no logo.

Reply:
Yes exactly like that:
<img width="351" alt="screen shot 2017-10-19 at 10 35 20 am" src="https://user-images.githubusercontent.com/19613940/31785208-4a4a87c8-b4b9-11e7-91cc-15665ff33ca6.png">

Reply:
@larsjohansen-okta can you confirm if this is the correct behavior for the API response?

Reply:
I think that's up to @johngronberg-okta to decide. The natural thing would be to either exclude the "logo" section fully or keep it as it is now, which is what you've shown above.

Reply:
We should be consistent with the behavior we have for other APIs.

Reply:
@larsjohansen-okta what is the consistent behavior? Seems to me that a ```logo.type: 'image/png'``` without an href is kind of weird, but you are the api expert
@manueltanzi-okta if api remains the same, this looks good. Otherwise, we just need to change this mock response. It will definitely be ideal if we can have our test response exactly the same as the api.